src/yaml/onlyplace4/17861-blam-signage-volume-1.yaml

```diff
@@ -0,0 +1,44 @@
+group: onlyplace4
+name: blam-signage-volume-1
```

Comment:
Perhaps you can manually rename this to vol01, as this is more in line with the existing content.